sphynx/python/llm_pandas_on_graph.py

```diff
@@ -0,0 +1,211 @@
+'''Generates Pandas code for a graph using OpenAI.'''
+import numpy as np
+import pandas as pd
+import os
+import sys
+import openai
+# Langchain is not on Conda yet. Run "pip install langchain".
+# https://github.com/hwchase17/langchain/issues/1271
+import langchain
+
+openai.api_key = os.getenv('OPENAI_API_KEY')
+
+
+examples = [
```

Comment:
@lacca0 This is the kind of thing I tried to describe in #348. We have some scenarios and we can automatically put them in a prompt (`function_def`) that we can tweak separately. The examples seem very important to me. Without them Codex generated me stuff like:
```python
  # YOUR CODE HERE
  #raise NotImplementedError()
```